### PR TITLE
Alerting: Unpausing a non-paused alert rule should not change status to Unknown

### LINKED ISSUE #21330: Unpausing a non-paused alert changes status to Unknown
**What happened**:
Unpausing a non-paused Alert changes the state to "unknown", adding an annotation next time the Alert executes.

**What you expected to happen**:
Sending an unpausing API call should trigger only if the Alert is paused. If should be ignored for non-paused alerts.

**How to reproduce it (as minimally and precisely as possible)**:
Send a '"paused": false' to a non-paused Alert and it will change the state to "unknown"

**Environment**:
- Grafana version: 6.5.2
- OS Grafana is installed on: Grafana Cloud

### PR DESCRIPTION
**Pull request by vikkyomkar**
**What this PR does / why we need it**:
This PR validates alert's current state before making any change which helps in avoiding issues due to
-> Pausing the already paused alert (or)
-> Un-pausing the already up-paused alert 

**Which issue(s) this PR fixes**:

Fixes #21330 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by vikkyomkar** [comment]
@torkelo @marefr @daniellee can you guys pls review this PR ?

File: pkg/api/alerting.go [hunk 1]
```diff
@@ -353,10 +353,12 @@ func NotificationTest(c *models.ReqContext, dto dtos.NotificationTestCommand) Re
 
 //POST /api/alerts/:alertId/pause
 func PauseAlert(c *models.ReqContext, dto dtos.PauseAlertCommand) Response {
+	const Pause = "paused"
```
Comment (torkelo) [inline pkg/api/alerting.go]: we have const for this in models.AlertStatePaused

**Review by torkelo — CHANGES_REQUESTED** [review]

File: pkg/api/alerting.go (hunk 1 above)
Comment (vikkyomkar) [reply pkg/api/alerting.go]: Hi @torkelo ...I have modified the code to refer const defined in models.

**Review by aknuds1 — APPROVED** [review]
LGTM!

**Comment by torkelo** [comment]
thanks for contributing!

**Pull request merged**